pyproject.toml

```diff
@@ -352,6 +352,9 @@ twine = "*"
 # Towncrier min version comes from #3425. Rationale unclear.
 towncrier = ">=18.6.0rc1"
 
+# Used for checking the Poetry lockfile
+tomli = ">=1.2.3"
+
```

Comment:
Note that `tomli` wasn't added to the lockfile; presumably it was already pulled in by something else?